### DISCUSSION TITLE
With Acro Mode the plane drifts down with an angle around 45degrees

### DISCUSSION THREAD

**Discussion by ghost** [iNavFlight/inav-configurator#1760] [General]
When Acro Mode is activated ( and this is the standard mode) you are not able to start the plane or fly with this mode.
The plane is drifting dramatically down with an angle around 45 degrees.
Seems the the elevator stick is pulled down extremly

With other modes you can fly, but not with acro mode

INAV 6.0 with Matek FC405TE and latest SW

**Comment by rts18** [comment]
Are you aware Acro Mode is not a LEVEL stabilized flight mode, as Angle and Horizon modes?
Acro Mode only provides attitude stabilization, with its strength being determined by the axis i-term gain.

In short, Acro mode is a dampened version of Manual mode, for fixed-wing pilots that prefer a smoother stick feel, when performing aerobatic flight.

**Comment by ghost** [comment]
you are correct, but with this kind of mode its not possible to fly the plane.
If you are switching to this mode, it always starts directly to fly down to the earth with around 45 degrees.
Even with the full activation of the elevator switch up, its not able to fly straight ahead, it always is flying  down
This happens also with the manual mode.
So with manual mode not able to fly, with other stabilized modes you can fly
This is strange I think

**Comment by MrD-RC** [comment]
If the problem is also in manual mode. The issue is with the physical setup of the plane. Hands off, it should fly reasonably straight in manual. 

I would check the CG and your control surfaces. Maybe check the outputs page in Configurator and the servo midpoints (if you have continuous servo trim enabled). This may also give clues. 

Rule of thumb. If it flies badly in manual. It is the plane. It would fly the same without the flight controller installed.

**Comment by ghost** [comment]
I fully agree to you.  I am  having experience of flying rc planes by round about 35 years now.
The CG is checked and fits.
I am wondering, because I am not able to handle the plane to fly straight ahead even pulling the elevator stick fully.
With any FC Mode it works (and there the FC is handling the elevator)
With manual handling of elevator it doesn't even with not with full amplitude
Thats why I am wondering....
